### ISSUE TITLE
Sample App is crashed: Caused by: kotlin.TypeCastException: null cannot be cast to non-null type pyxis.uzuki.live.mediaresizer.data.VideoResizeOption

### ISSUE THREAD

**Issue by bhaveshjabuvani-credencys** [WindSekirun/MediaResizer#1]
I download apk from link given in descriptoin. I am trying to capture image at that time it is crash:
Exception: 
```
Process: pyxis.uzuki.live.mediaresizersample, PID: 30368

java.lang.RuntimeException: Failure delivering result ResultInfo{who=android:fragment:1, request=0, result=-1, data=Intent { act=inline-data (has extras) }} to activity {pyxis.uzuki.live.mediaresizersample/pyxis.uzuki.live.mediaresizersample.activity.JavaActivity}: kotlin.TypeCastException: null cannot be cast to non-null type pyxis.uzuki.live.mediaresizer.data.VideoResizeOption
add dependenciesat android.app.ActivityThread.deliverResults(ActivityThread.java:4094)
add dependenciesat android.app.ActivityThread.handleSendResult(ActivityThread.java:4137)
add dependenciesat android.app.ActivityThread.-wrap20(ActivityThread.java)
add dependenciesat android.app.ActivityThread$H.handleMessage(ActivityThread.java:1529)
add dependenciesat android.os.Handler.dispatchMessage(Handler.java:102)
add dependenciesat android.os.Looper.loop(Looper.java:154)
add dependenciesat android.app.ActivityThread.main(ActivityThread.java:6123)
add dependenciesat java.lang.reflect.Method.invoke(Native Method)
add dependenciesat com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:867)
add dependenciesat com.android.internal.os.ZygoteInit.main(ZygoteInit.java:757)

Caused by: kotlin.TypeCastException: null cannot be cast to non-null type pyxis.uzuki.live.mediaresizer.data.VideoResizeOption
add dependenciesat pyxis.uzuki.live.mediaresizer.data.ResizeOption$Builder.build(ResizeOption.kt:61)
add dependenciesat pyxis.uzuki.live.mediaresizersample.activity.JavaActivity.processImage(JavaActivity.java:99)
add dependenciesat pyxis.uzuki.live.mediaresizersample.activity.JavaActivity.resultProcess(JavaActivity.java:80)
add dependenciesat pyxis.uzuki.live.mediaresizersample.activity.JavaActivity.lambda$clickCamera$0$JavaActivity(JavaActivity.java:46)
add dependenciesat pyxis.uzuki.live.mediaresizersample.activity.JavaActivity$$Lambda$0.invoke(Unknown Source)
add dependenciesat pyxis.uzuki.live.richutilskt.utils.RPickMedia$pickFromCamera$1.invoke(RPickMedia.kt:66)
add dependenciesat pyxis.uzuki.live.richutilskt.utils.RPickMedia$pickFromCamera$1.invoke(RPickMedia.kt:23)
add dependenciesat pyxis.uzuki.live.richutilskt.utils.RPickMedia$ResultFragment.onActivityResult(RPickMedia.kt:189)
add dependenciesat android.app.Activity.dispatchActivityResult(Activity.java:6955)
add dependenciesat android.app.ActivityThread.deliverResults(ActivityThread.java:4090)
```

**Comment by WindSekirun** [comment]
Thanks for reporting.

i need some information to solve this issue.

1. can you write down reproduce process?
2. can you provide full log of crash?

by default, all variable of VideoResizeOption except MediaFormatStrategy can not be null.

**Comment by WindSekirun** [comment]
Oh, i don't see 'edited'.

so, i will check my sample project, and i will upload new apk file.

**Comment by WindSekirun** [comment]
I uploaded new sample apk in 101c9f068bb1308686bda703bf9a43f639eb4b84. please check.

I try to find reproduce process to solve this issue, but my device is operated successfully.
but i fix crash issue related your issue in a874421cd7760b3b29dcee06fce0bab0e8868fea. 
i hope this commits help you.

**Comment by WindSekirun** [comment]
I made another commit which make nullable VideoResizingOption, ImageResizingOption in ResizeOption in 124c036aabf321f8e0bf324d8ac7d6f0e4befeca.
